configs/infer_config.yaml

```diff
@@ -8,44 +8,73 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
+### Data Pre-processing parameters
+## Generate intervals files
+intervalsize: 50000
```

Comment:
there's a bit of inconsistency in our naming scheme where some options are `wordword` and some are `word_word`, like `intervalsize` vs `batch_size`. We should make the consistent (and I personally like the latter better)